kolibri/plugins/device/assets/src/constants.js

```diff
@@ -71,3 +71,8 @@ export const MeteredConnectionDownloadOptions = {
   DISALLOW_DOWNLOAD_ON_METERED_CONNECTION: 'DISALLOW_DOWNLOAD_ON_METERED_CONNECTION',
   ALLOW_DOWNLOAD_ON_METERED_CONNECTION: 'ALLOW_DOWNLOAD_ON_METERED_CONNECTION',
 };
+
+export const Presets = Object.freeze({
+  FORMAL: 'formal',
+  NONFORMAL: 'nonformal',
+});
```

Comment:
Hi @Jaspreet-singh-1032. There looks to be similar constants in `kolibri/plugins/setup_wizard/assets/src/constants.js`, so I think its better to use them instead for maintenance purposes!

Reply:
Hi @akolson, I have made some changes. Have a look and let me know if it needs any other changes.
Thanks.

Reply:
Maybe a clean up of this? Looks like `Presets` has been moved to `kolibri/core/assets/src/constants.js` so nolonger necessary in here